Makefile

```diff
@@ -296,28 +298,26 @@ kind-extensions-clean:
 
 kind-operator-up: $(KIND) $(KUBECTL) $(HELM) $(YQ) $(KUSTOMIZE)
 	./hack/kind-up.sh \
-		--cluster-name gardener-operator-local \
-		--path-kubeconfig $(REPO_ROOT)/example/gardener-local/kind/operator/kubeconfig \
+		--cluster-name $(CLUSTER_NAME) \
+		--path-kubeconfig $(KIND_KUBECONFIG) \
 		--path-cluster-values $(REPO_ROOT)/example/gardener-local/kind/operator/values.yaml \
 		--with-lpp-resize-support $(DEV_SETUP_WITH_LPP_RESIZE_SUPPORT)
-	mkdir -p $(REPO_ROOT)/dev/local-backupbuckets/gardener-operator
```

Comment:
Why this line was removed?

Reply:
Please see [here](https://github.com/gardener/gardener/pull/9763#discussion_r1610414345).